crates/recursor/src/recursor.rs

```diff
@@ -19,17 +19,21 @@ use crate::recursor_dns_handle::RecursorCacheMetrics;
 use crate::{
     DnssecPolicy, Error,
     proto::{
+        ProtoError,
```

Comment:
CI is calling this out. Does it need to go back inside the `#[cfg(feature = "__dnssec")]` gate?

```
error: unused import: `ProtoError`
  --> crates/recursor/src/recursor.rs:22:9
   |
22 |         ProtoError,
   |         ^^^^^^^^^^
```